cmd/api/src/config/config_test.go

```diff
@@ -24,66 +24,156 @@ import (
 )
 
 func TestSetValuesFromEnv(t *testing.T) {
```

Comment:
I updated this test to use subtests to make it easier to see which test cases are being used